test/e2e/aigateway_e2e_test.go

```diff
@@ -0,0 +1,179 @@
+/*
+Copyright 2025 Agentic Layer.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/agentic-layer/ai-gateway-litellm/test/utils"
+)
+
+// namespace where the project is deployed in
+const namespace = "ai-gateway-litellm-system"
+
+var _ = Describe("AiGateway", Ordered, func() {
+
+	BeforeAll(func() {
```

Comment:
You can also adapt the e2e_suite as in agent-gateway to have this setup code for all e2e test files.